ehrql/tables/beta/tpp.py

```diff
@@ -680,31 +762,31 @@ class opa_proc(EventFrame):
     opa_ident = Series(
         int,
         constraints=[Constraint.NotNull()],
-        description="TODO",
+        description="Primary key used to join to the other OPA tables",
     )
     primary_procedure_code = Series(
         OPCS4Code,
-        description="TODO",
+        description="The OPCS classification of interventions and procedures code which is used to identify the primary patient procedure carried out.",
     )
     primary_procedure_code_read = Series(
         CTV3Code,
-        description="TODO",
+        description="The read coded clinical terms code which is used to identify the primary patient procedure carried out.",
     )
     procedure_code_1 = Series(
         OPCS4Code,
         description="TODO",
```

Comment:
question: What is `procedure_code_1`? It's not in the [NHS database search](https://data.england.nhs.uk/ncdr/search/column/?q=procedure_code_1)

Reply:
Is this a duplicate of the primary procedure code? Or something else?